server/dive_server/crud_rpc.py

```diff
@@ -377,6 +378,31 @@ def postprocess(
 
     if not skipJobs and not isClone:
         token = Token().createToken(user=user, days=2)
+
+        # extract ZIP Files if not already completed
+        zipItems = Folder().childItems(
+            dsFolder,
+            filters={
+                "lowerName": {"$regex": constants.zipRegex},
+                f"meta.{constants.ZipFileExtractedMarker}": {'$exists': False},
```

Comment:
The item is moved to `source/`, so looking for items where the metadata marker is false has no additinal selection power. Can this be removed?  Or is it just here for defensiveness?

If it's here for defensiveness, I think I'd prefer logic that raises an exception if an impossible condition is encountered.  Maybe remove this from the query, and then `for item in zipItems: if marker is set, raise exception`

Then it will show up as a 500 in the logs and in sentry.

Reply:
It's looking for files where the marker doesn't exists, not where it is false but your point is correct.  It's a remnant from before I moved stuff into a source folder.  Maybe I should just remove it.  There is no current use for it, any justification is based on future uses of the /source folder.